main.py

```diff
@@ -1,7 +1,7 @@
 import random
 
 
-def add_git_branch():
+def add_new_git_branch():
     return "https://github.com/AssafBrown92/mego_game.git"
 

```

Comment:
אני לא ממש מבין מה הפונקציה הזו עושה.
בנוסף, פעם הבא אנא תן תיאור קטן לקומיט שלך. אם אין לך איך לתאר אותו כתוב רק minor

Reply:
nothing to interesting, just returns the project's URL, 
no need to actauly implment it